broker/httpd/auth.go

```diff
@@ -21,6 +23,28 @@ func randomStringGeneration() (string, error) {
 	return base64.URLEncoding.EncodeToString(bytes), nil
 }
 
+func getRedirDestination(r *http.Request) string {
+	destinationPath := "/"
+	if !(r.Method == "GET" || r.Method == "POST") {
```

Comment:
I think inverting a composite condition makes the logic more obscure. Why not:
`if r.Method != "GET" && r.Method != "POST {`
Alternatively, use a `case` statement.

Reply:
I have always found x!=b && x!=a more confusing than !(x==b || x==a).

Reply:
Frankly, both are fairly explicit in the conveyed logic, and readability may depend more on the font you are viewing this in, rather than the construction, so I'd say it's a nit. As long as the condition is consistent with other similar pieces of the code either way is fine.

Reply:
One disadvantage of composite condition invertion I can think of though is if the condition will become more complicated in the future it can quickly become much less readable/hard to comprehend in such form.